project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.0-RC4
```

Comment:
I believe this was not intended to get here, right?

Reply:
Oops. Good catch.